WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -350,7 +358,12 @@ extension OrderListViewController: SyncingCoordinatorDelegate {
         // Configure header container view
         let headerContainer = UIView(frame: CGRect(x: 0, y: 0, width: Int(tableView.frame.width), height: 0))
         headerContainer.addSubview(topBannerView)
-        headerContainer.pinSubviewToSafeArea(topBannerView)
+        NSLayoutConstraint.activate([
+            headerContainer.leadingAnchor.constraint(equalTo: topBannerView.leadingAnchor),
+            headerContainer.trailingAnchor.constraint(equalTo: topBannerView.trailingAnchor),
+            headerContainer.safeAreaLayoutGuide.topAnchor.constraint(equalTo: topBannerView.topAnchor),
+            headerContainer.safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: topBannerView.bottomAnchor),
+        ])
```

Comment:
Q: It's not clear to me why this change. Why we don't want to maintain leading and trailing anchors to the safe area?

Reply:
I want the Order list to extend to all edges - this should be safe to do when all its subviews are anchored to safe area. I fixed the issue with the layout of the footer spinner view as well as the filters bar that you captured above in 09895c0523eb83d54bcff15cf709c7f49b84658a.